Lab_1.ipynb

```diff
@@ -1,6 +1,700 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "7163cfd5-743f-44bc-a3da-4e8928d6dcba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "measurement_value = 1\n",
+    "\n",
+    "mu = 8"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e732a0d3-a5d3-4788-896f-07b55edb664b",
+   "metadata": {},
+   "source": [
+    "This time I'm integrating the function to the left of my value, so using *stats.poisson.cdf* will give me the correct probability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "07714838-62e0-429e-948e-0d67d35f7a8f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "There is a 1-in-331 chance the background alone produces a value as or more signal-like than my measurement value.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# find poisson probability\n",
+    "\n",
+    "prob = stats.poisson.cdf(measurement_value, mu)\n",
+    "\n",
+    "print(\"There is a 1-in-\"+str(round(1/prob))+\n",
+    "      \" chance the background alone produces a value as or more signal-like than my measurement value.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc91392b-ec58-47cf-9ffc-e4aaef1d4632",
+   "metadata": {},
+   "source": [
+    "Like before, I can use *stats.norm.ppf* to reverse-integrate a Gaussian distribution based on my calculated probability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "2bc6def5-3caf-4bed-a8c1-dd2cbb1d5e4d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "A successful detection at a measurement value of 1 can be reported with a certainty of 2.7σ.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# find Xσ\n",
+    "\n",
+    "sigma = stats.norm.ppf(prob, loc=0, scale=1)\n",
+    "\n",
+    "print(\"A successful detection at a measurement value of \" +str(measurement_value)+ \n",
+    "      \" can be reported with a certainty of \" +str(abs(round(sigma,1)))+ \"σ.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4a80ac5-6417-4ea1-a205-3427d2ad169b",
+   "metadata": {},
+   "source": [
+    "Since the probabilities and sigma values are discreet, a Xσ value of 2.7 doesn't make much sense. This must either be rounded down to 2 or up to 3, neither of which provide the most accurate value. The distribution *can* have a non-integer mean, however, as the mean is a single calculation and can lie between discreet data points. The time (or space) interval of the Poisson distribution changes continuously, but the number of events is discreet. "
```

Comment:
I liked the last sentence about how the Poisson distribution can change continuously vs. discretely, but I think it may also be good to include what often used parameters of it change continuously vs. discretely other than the mean and sigma (like standard deviation for example).